README.md

```diff
@@ -350,6 +350,28 @@ limit the commands to the specific dataset!_
    1. Ask AWS support to remove the delete markers returned by
       `aws s3api list-object-versions --bucket=linz-geostore --prefix=DATASET_TITLE/ | jq .DeleteMarkers`.
 
+## Geostore Release
+
+### Versioning and Frequency
+
+We aim to release at the end of each agile sprint (fortnightly), or whenever required (e.g. bugfix,
```

Comment:
Do we need to mention that publishing a release triggers a prod deployment via GH actions?